spec/index.bs

```diff
@@ -538,6 +538,10 @@ If an object is successfully created, it is <dfn dfn>valid</dfn> at that moment.
 An [=internal object=] may be <dfn dfn>invalid</dfn>.
 It may become [=invalid=] during its lifetime, but it will never become valid again.
 
+Issue: Consider separating "invalid" from "destroyed".
+This would let validity be immutable, and only operations involving devices,
```

Comment:
Destroying the device would also prevent submitting work on the queue for example.